scene/animation/animation_mixer.h

```diff
@@ -377,6 +390,12 @@ class AnimatedValuesBackup : public RefCounted {
 public:
 	void set_data(const HashMap<NodePath, AnimationMixer::TrackCache *> p_data) { data = p_data; };
 	HashMap<NodePath, AnimationMixer::TrackCache *> get_data() const { return data; };
+
+	~AnimatedValuesBackup() {
+		for (KeyValue<NodePath, AnimationMixer::TrackCache *> &K : data) {
+			memdelete(K.value);
```

Comment:
You might need a guard here to prevent a crash in a case where the `TrackCache *` is a nullpointer. Something like

```cpp
if (K.value) {
    memdelete(K.value);
}
```
or 
```cpp
ERR_FAIL_NULL(K.value);
memdelete(K.value);
```

Reply:
This won't help if the pointer is freed (points to invalid, non-null, memory).